docs/src/integrals.md

```diff
@@ -0,0 +1,76 @@
+# Integrals and Orbitals
+
+## IntegralHelper
+
+The `Fermi.Integrals.IntegralHelper` object is designed to provide easy access to molecular 
+integrals from Julia. 
+
+### Creation
+To create an `IntegralHelper`, you can simply call,
+
+    Fermi.Integrals.IntegralHelper()
+
+and one will be created based on the information in `Fermi.CurrentOptions`, which you may recall are set with `@set`. 
+If you want more control, you can pass keyword arguments for the `Molecule` and `OrbDict` to be used,
+
+    Fermi.Integrals.IntegralHelper(mol=mymol,orbs=myorbs)
+
+### Use
+Once an `IntegralHelper` object is created , integrals are simply accessed as,
+
+    helper["S"] #AO basis overlap
+    helper["T"] #AO basis kinetic
+    helper["V"] #AO basis nuclear attraction
+    helper["\mu"] #AO basis ERI
```

Comment:
tab this mu

Reply:
in .md files tabcomplete doesn't work :(